lib/hammer_cli_katello/content_view_version.rb

```diff
@@ -57,6 +55,17 @@ class InfoCommand < HammerCLIKatello::InfoCommand
           field :name, _("Name")
           field :label, _("Label")
         end
+
+        field :filters_applied, _("Has Applied Filters")
+        collection :applied_filters, _("Filters Applied") do
+          collection :applied_filters, _("") do
```

Comment:
In case the structure will always be:
```ruby
{
  #...
  "filters_applied": true/false,
  "applied_filters": {
    "applied_filters": [
      {
        #...
      }
    ]
  }
}
```
With the first `applied_filters`  being redundant, then we can do:
```suggestion
        from :applied_filters,  do
          collection :applied_filters, _("Filters Applied"), hide_blank: true, hide_empty: true do
```

Reply:
I was in a meeting yesterday with a couple people to address this, and the best we could come up with was the nested collection solution which slightly butchers the 'true' case but handles the 'false' and absent cases well for `--include-applied-filters`. Admittedly none of us were experts in this area.

Reply:
Chris and I were able to fix this with a hide-blank on Dependency Solving. Thank you for the help with putting hide flags on the collection :)